docs/installation.md

```diff
@@ -0,0 +1,28 @@
+---
+id: installation
+title: Installation
+---
+
+The goal for Traduttore is to make it as easy as possible to supercharge your WordPress internationalization workflow.
+
+## System Requirements
+
+Traduttore is a WordPress plugin that sits on top of the [GlotPress](https://glotpress.org/) plugin. That means you need to have both WordPress and GlotPress installed.
+
+GlotPress is available as a WordPress plugin through the plugin directory. Installing it is as simple as searching for “GlotPress” and installing it. After activating the plugin, GlotPress can be accessed via `https://<home_url>/glotpress/`.
+
+Traduttore requires at least PHP 7.1, while the [Traduttore Registry](https://github.com/wearerequired/traduttore-registry) also supports PHP 7.0.
+
+Traduttore requires [WP-CLI](https://wp-cli.org/) 2.0 or newer to be installed on the server. [Learn more about the available CLI commands](cli.md).
+
```

Comment:
We could add the Slack plugin requirement here.

Reply:
Adding a separate doc for that instead.